users/users-model.js

```diff
@@ -137,4 +137,6 @@ function edituserProfile(Userid, body){
   .returning('id')
   .where({id:Userid})
   .update(body, "id")
-}
\ No newline at end of file
+}
+
+myUndefinedFunction();
```

Comment:
What is this?

Reply:
the sentry documentation called for this anonymous function to be placed in the code so that it can throw an exception